src/CSH/Eval/Frontend/Data.hs

```diff
@@ -34,6 +35,7 @@ data EvalFrontend = EvalFrontend
                   { getStatic :: Static
                   , getConfig :: ServerCmd
                   , getCache  :: Cache
+                  , getLogger :: Logger
```

Comment:
It might makes sense to call this `getFrontendLogger`, especially if it's going to conflict with top level declarations from the logging library